src/styles/catalog.scss

```
@@ -0,0 +1,123 @@
+.catalog {
+  display: flex;
+  margin: 0 auto;
+  flex-wrap: wrap;
+  width: $catalog-width;
+  padding: 51px 0 0;
```

bottom paddings should be the same as top (by the task)
```suggestion
  padding: 51px 0;
```